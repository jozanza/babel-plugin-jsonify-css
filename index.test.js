import test from 'ava';
import { transformFileSync } from 'babel-core';

test(t => {
  const { code } = transformFileSync('./main.js', {
    babelrc: false,
    plugins: ['./index.js']
  });
  t.snapshot(code);
});
