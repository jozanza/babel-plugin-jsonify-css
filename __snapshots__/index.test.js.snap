exports[`[anonymous] 1`] = `"\"var css = {\\n  \'.foo\': {\\n    \'width\': \'100%\'\\n  },\\n  \'@media(min-width: 100px; max-width: 1000px)\': {\\n    \'.foo\': {\\n      \'bar\': \'88px\'\\n    }\\n  }\\n};\\nvar rule = [{\\n  \'.foo\': {\\n    \'width\': \'100%\'\\n  }\\n}];\\nvar media = [{\\n  \'@media(min-width: 100px; max-width: 1000px)\': {\\n    \'.foo\': {\\n      \'bar\': \'88px\'\\n    }\\n  }\\n}];\\nvar keyframes = [[\'bounce\', {\\n  \'0%,50%\': {\\n    \'blah\': \'.5\'\\n  },\\n  \'to\': {\\n    \'lol\': \'100vh\'\\n  }\\n}]];\\nvar fontFace = [{\\n  \'fontFamily\': \'Open Sans\',\\n  \'fontStyle\': \'normal\',\\n  \'fontWeight\': \'400\',\\n  \'src\': \'local(\\\\\'Open Sans\\\\\'),\\\\n       local(\\\\\'OpenSans\\\\\'),\\\\n       url(\\\"https://fonts.gstatic.com/s/...\\\")\'\\n}];\\nvar charset = [{\\n  \'@charset\': \'\\\"UTF-8\\\"\'\\n}];\\nvar raw = \'@charset \\\"UTF-8\\\";\\\\n\\\\n.foo {\\\\n  width: 100%;\\\\n}\\\\n\\\\n@keyframes bounce {\\\\n  0%, 50% { blah: .5; }\\\\n  to { lol: 100vh; }\\\\n}\\\\n\\\\n@media(min-width: 100px; max-width: 1000px) {\\\\n  .foo {\\\\n    bar: 88px;\\\\n  }\\\\n}\\\\n\\\\n@font-face {\\\\n  font-family: Open Sans;\\\\n  font-style: normal;\\\\n  font-weight: 400;\\\\n  src: local(\\\\\'Open Sans\\\\\'),\\\\n       local(\\\\\'OpenSans\\\\\'),\\\\n       url(\\\\\'https://fonts.gstatic.com/s/...\\\\\');\\\\n}\\\\n\';\""`;
